### ISSUE TITLE
Create GitHub Pages site

### ISSUE THREAD

**Issue by jcbhmr** [jcbhmr/devcontainer-features#3]
- [ ] Turn on GitHub Pages from actions
- [ ] Add GitHub Action to deploy to pages
- [ ] Make the page itself

The criteria for the pages site are:
- [ ] Have an index page of all the features
- [ ] Have a unique URL for each feature
- [ ] Have pretty URLs (use name/index.html instead of name.html)
- [ ] Each individual feature page has an options table
- [ ] Each individual feature page has an example copy-paste-able
- [ ] Feat page has footer with "Feature doesn't work? Open an issue"
- [ ] Website has Troubleshooting guide page for users (NOT devs so it's not on the dev wiki)
- [ ] Feat page has footer "Need help? Find us on Gitter"

_Some of these are inspired by https://github.com/devcontainers-contrib/features/discussions/290_

Some options for making the website itself:
- PHP + (Wget or HTTrack)
- VuePress https://v2.vuepress.vuejs.org/guide/getting-started.html
- Eleventy https://www.11ty.dev/
- Astro https://docs.astro.build/en/getting-started/

More SSGs: https://jamstack.org/generators/

Basically the site needs to have a template for the index that for-loops through each feature as a card or similar, then have a per-feature `[id].md` or similar that generates a table for each feature.

**Comment by jcbhmr** [comment]
Decided to focus on feature content instead of documentation right now.

**Comment by jcbhmr** [comment]
I got something working on my own repo!
https://github.com/jcbhmr/devcontainer-features/tree/main/docs
https://jcbhmr.github.io/devcontainer-features/

Something similarish for this repo in the future maybe? 🤔

cc @eitsupi I'm pinging you to show off that **I did a thing** that **actually worked** and I think is pretty cool. ❤️

**Comment by eitsupi** [comment]
I think this is a great step forward!

Perhaps a site search should be implemented.
